pages/data.qmd

```diff
@@ -0,0 +1,70 @@
+
+## Missingness
+
+There is substantial missingness at the county level, especially for rural and lower-population locations.
+
+Missingness has varied a bit since reporting began in late 2022. As of January 2025, NSSP does not report any data at the county level for the following states: Alaska, Alabama, Arkansas, Arizona, California, Florida, Missouri, North Dakota, New Hampshire, Ohio, Pennsylvania, Rhode Island, South Dakota, and Washington. 
+
+Of these, no county-level data is available since reporting began for Alaska, Alabama, Arkansas, Arizona, California, Florida, Missouri, North Dakota, New Hampshire, Ohio, South Dakota, and Washington. 
+
+**We are open to working with public health partners from jurisdictions with no reported NSSP county-level data if they are able to facilitate access to an alternative source of sub-state-level data.**
+
+## Accessing NSSP data
+
+NSSP data can be accessed through the [epidata API](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/nssp.html) using `epidatr` (recommended) and `covidcast` R packages.
+
+A vignette for using `epidatr` is available on [CRAN](https://cran.r-project.org/web/packages/epidatr/vignettes/epidatr.html).
+
+```{r, message = FALSE}
+library(epidatr)
+library(covidcast)
+```
+
+Although the recommended way to pull data is with `epidatr`, we start by illustrating county-level data using the `covidcast` package's plotting functionality. The following plot shows reporting counties as of Jan. 12, 2025. Recall all counties in an HSA will have the same value as the parent HSA. 
+
+```{r}
+nssp_county <- covidcast::covidcast_signal("nssp", "pct_ed_visits_influenza",
+                                geo_type = "county",
+                                time_type = "week")
+```
+
+```{r}
+plot(nssp_county, time_value = "2025-01-12")
+```
+
+The following code uses the `epidatr` package to return county-level data, where `geo_value` is the FIPS code of the county. Again, the values returned for an individual county are really those of the parent HSA.
+
+```{r, message = FALSE}
+epidatr::pub_covidcast(source = "nssp", signals = "pct_ed_visits_influenza", geo_type = "county", geo_values = "25015", time_type = "week")
+```
+
+## Why use NSSP HSA data?
+
+Metro-level disease trends and forecasts may differ markedly from state and national trends. Building the capacity to forecast at the metro-level may be crucial in times of outbreak planning and response.  
+
+NSSP HSA-level data are **publicly available**, making it quicker and easier to expand forecasting efforts to additional jurisdictions that report these data. 
+
+A more detailed visual breakdown of HSAs for each state can be found [here](https://raw.githubusercontent.com/mmkerr/draft/main/all_states_faceted.pdf). This document also provides line plots for each state illustrating the difference between the percentage of ED visits for the state (grey line) vs. each individual HSA in that state (orange line). 
```

Comment:
Sorry I didn't catch this the first time. I think this image should be part of the PR. See example from the hub-dashboard-template for an example of including images: https://github.com/hubverse-org/hub-dashboard-template/tree/main/pages/img

Ideally, it would be a PNG or JPEG file, not a PDF, as those image types are better for loading on webpages.

While it's hard to locate, I did see that there was an error in the Site Building actions:
https://github.com/reichlab/metrocast-dashboard/actions/runs/17685601766/job/50269612553#step:8:20

It's a bug that this doesn't appear more centrally in the build actions.